src/TwigExtension.php

```diff
@@ -676,7 +676,8 @@ public function current($content)
             $linkToCheck = (string) $content;
         }
 
-        $requestedUri    = reset(explode('?', $this->app['request']->getRequestUri()));
+        $uriFromRequest = $this->app['request']->getRequestUri();
```

Comment:
I want to refactir this soon, probably using `parse_url()`. For now, fixing bugs is fine. :-)